security/middleware.py

```diff
@@ -22,6 +22,8 @@
 from ua_parser import user_agent_parser
 
 logger = logging.getLogger(__name__)
+profiling_logger = logger.getLogger("profiling")
```

Comment:
we want direct profiling-specific debug messages to a dedicated logger as I understand. right? Why we using `logger` here instead of defining the `profiling` logger using `logging`?

```
profiling_logger = logging.getLogger("profiling")
```

Reply:
ohhhh good catch, bad typo on my part x.x